src/backend/MeshRecordComponent.cpp

```diff
@@ -44,10 +44,16 @@ MeshRecordComponent::read()
     Attribute a = Attribute(*aRead.resource);
     if( *aRead.dtype == DT::VEC_FLOAT )
         setPosition(a.get< std::vector< float > >());
+    else if( *aRead.dtype == DT::FLOAT )
+        setPosition(std::vector< float >(a.get< float >()));
```

Comment:
This actually initializes a vector with ```static_cast< std::vector< float >::size_type >(a.get< float >())``` default-initalized elements.
The intended behaviour is realized with
```cpp
setPosition(std::vector< float >({a.get< float >()}));
```

Reply:
@C0nsultant do you want to update the branch with it?

Reply:
Will do before merging.